tpl/partials/partials.go

```diff
@@ -72,6 +73,10 @@ func (ns *Namespace) Include(name string, contextList ...interface{}) (interface
 			b := bp.GetBuffer()
 			defer bp.PutBuffer(b)
 
+			if ns.deps.Cfg.GetBool("templateMetrics") {
+				defer ns.deps.Metrics.MeasureSince(templ.Name(), time.Now())
+			}
+
 			if err := templ.Execute(b, context); err != nil {
```

Comment:
It looks to me that every `MeasureSince` is wrapping a call to `templ.Execute` (or similar). `templ` is a Hugo interface, and we should put the `MeasureSince` directly in the `Execute` method (there are some embedding going on there currently, but that is easy to work around with some delegation. This will not only be much cleaner, but it will also solve the `GetBool` issue, as you can set that as a config once when you create the template harness.